src/Navigation.tsx

```diff
@@ -20,19 +20,14 @@ import { Builder } from "features/builder/Builder";
 import { AuthMachineState } from "features/auth/lib/authMachine";
 import { ZoomProvider } from "components/ZoomProvider";
 import { LoadingFallback } from "./LoadingFallback";
-
```

Comment:
This was removed due to a potential issue/bug when loading these lazily. On rare occasions these bundles are not loaded correctly. This presents a potential issue when caching static assets. I have removed for now as there are known issues with lazy loaded resources. 

The PWA performance gains have not really been impacted here.